config.yaml

```diff
@@ -1,70 +1,125 @@
-prometheus_url: http://host.docker.internal:9000
+prometheus_url: http://localhost:9090
```

Comment:
@WeepingClown13 revert back the url to http://host.docker.internal:9000 to pass the testcase